Fix DM commands

pleasantone opened the pull request:
Unable to send a command to SlackRTM via DM (needed for private admin only commands).

Admins need to be able to send private commands to SlackRTM via DM, but the early check for
is the channel synced was cutting them out.

endofline commented:
we've moved over to v3.0 branch already and fixed this via commit 2a16a5a

on the other hand, this is actually a better fix, rather than slackrtm v2 actually listening to every single channel

pleasantone commented:
Oh crap. Thank you.

> On Apr 10, 2017, at 6:07 PM, Innocent Bystander <notifications@github.com> wrote:
> 
> we've moved over to v3.0 branch already and fixed this via commit 2a16a5a
> 
> on the other hand, this is actually a better fix, rather than slackrtm v2 actually listening to every single channel
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

endofline commented:
will leave this branch for 7 days before deleting in case there is any new issues arising from this merge
